src/test/java/com/cappella/csv/CsvClientTests.java

```diff
@@ -0,0 +1,165 @@
+package com.cappella.csv;
+
+import java.beans.Transient;
+import java.io.InputStream;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Locale;
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.commons.csv.CSVRecord;
+import com.cappella.model.Problems;
+import com.cappella.model.TaskData;
+import com.cappella.model.SubTask;
+
+class CsvClientTests {
```

Comment:
This should be a @SpringBootTest managed test.

Reply:
similar to the other unit test class - this caused a ton of errors initially.  Maybe it's worth a pair programming session.